sdk/go/pulumi-language-go/main.go

```diff
@@ -64,6 +64,9 @@ import (
 	"github.com/pulumi/pulumi/pkg/v3/codegen/schema"
 )
 
+// TODO test setting engineAddress from server in tests
+// TODO document PULUMI_DEBUG_LANGUAGES and PULUMI_DEBUG_PROVIDERS
```

Comment:
Still TODO?

Reply:
nope not anymore now that it is reviewed